server/test/tuist/slack/installation_test.exs

```diff
@@ -0,0 +1,170 @@
+defmodule Tuist.Slack.InstallationTest do
+  use TuistTestSupport.Cases.DataCase
```

Comment:
I'd add `async: true` to all the tests in the PR.

Reply:
Added `async: true`.